config/environments/development.rb

```diff
@@ -74,4 +74,8 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Allow the debugger to print to the console when running rails server under
+  # vagrant.
+  config.web_console.whitelisted_ips = '10.0.2.2'
```

Comment:
You can indeed allow a range. https://github.com/rails/web-console#configuration